main.py

```diff
@@ -169,6 +215,8 @@ def __iter__(self):
         rand_y = random.uniform(-1, 1) #nosec
         new_ki = creature.Ki + (rand_y * config.gen_mutation_coef)
         new_ki = max(0, min(new_ki, config.gen_max_ki))
+        if not validate_stability(new_kp, new_ki):
```

Comment:
Same as above:
In this implementation, if a result of a mutation is not stable, we are slowly reducing kp and ki to make it fit to the equation (P+I)^2 < 4I. Is this ok? Maybe we should introduce a param that will let user pick wether we want to do that or just drop creature (like mutation_unstable_policy = [drop, reduce]).

Reply:
Can we retry generating different random pair of parameters when they are not stable? I think if we drop them we can get in the deadlock where we'll not have any stable pair in a given mutation